source/localizable/models/relationships.md

```diff
@@ -43,7 +43,7 @@ import Model from 'ember-data/model';
 import { belongsTo } from 'ember-data/relationships';
 
 export default Model.extend({
-  post: belongsTo('post')
+  blogPost: belongsTo('blog-post')
```

Comment:
Yeah I guess it depends on whether we consider each paired example as isolated from the others and you can vary up file names to illustrate different points, or if the re-use of the same same example files throughout a chapter is illustrative by telling an implied narrative.

I think there is a lot of value to having this kind of example, to the point where changing all instances may be worth it if we think it's important to be consistent throughout the page. But I think it could be alright to vary up file names between individual examples.